catalog/spatial/registry/registry-ebrim-transformer/pom.xml

```diff
@@ -28,12 +28,22 @@
         <dependency>
             <groupId>org.jvnet.jaxb2_commons</groupId>
             <artifactId>jaxb2-basics-runtime</artifactId>
-            <version>0.6.0</version>
+            <version>0.11.0</version>
         </dependency>
         <dependency>
             <groupId>org.jvnet.ogc</groupId>
             <artifactId>ogc-tools-gml-jts</artifactId>
             <version>1.0.3</version>
+            <exclusions>
```

Comment:
❓Do we ever want to use `ogc-tools-gml-jts` by itself with the `jts` and `gml-v_3_1_1-schema` it pulls in? If not, should we consider adding a `dependencyManagement` element for it in one of the top level poms and put the exclusions in there instead of repeating them everywhere?

Reply:
I'm not sure. Probably not, but I don't know for a fact.